packages/desktop-electron/index.ts

```diff
@@ -54,6 +55,37 @@ if (!isDev || !process.env.ACTUAL_DATA_DIR) {
 let clientWin: BrowserWindow | null;
 let serverProcess: UtilityProcess | null;
 
+const createOAuthServer = async () => {
+  const port = 3010;
+  console.log(`OAuth server running on port: ${port}`);
+
+  return new Promise<{ url: string; server: Server }>(resolve => {
+    const server = createServer((req, res) => {
+      const query = new URL(req.url || '', `http://localhost:${port}`)
+        .searchParams;
+
+      const code = query.get('token');
+      if (code && clientWin) {
+        clientWin.loadURL(`http://localhost:3001/openid-cb?token=${code}`);
```

Comment:
I've just confirmed this: 

![image](https://github.com/user-attachments/assets/ba6eeeb3-9308-4d2a-ab1a-dcfbda7a8836)

localhost:3001 isn't running when running in prod mode. This will need to account for the running serverurl. In prod it should be app://actual I think

Reply:
ops, I messed this up. I will change it for the app url

Reply:
I'm trying to build electron to test it, but my environment is not ready for this (idk what's missing).

but I guess this would be the change, right?

```typescript
        if (isDev) {
          clientWin.loadURL(`http://localhost:3001/openid-cb?token=${code}`);
        } else {
          clientWin.loadURL(`app://actual/openid-cb?token=${code}`);
        }
```

Reply:
Probably. You can test by downloading the artifact on this PR.

Reply:
That solved it 🥳

Reply:
I was downloading the artifact to test too just in case

Reply:
working on windows build too 🎊